platform/features/imagery/src/controllers/ImageryController.js

```diff
@@ -75,50 +92,133 @@ define(
                         .telemetry
                         .getValueFormatter(metadata.valuesForHints(['image'])[0]);
                     this.unsubscribe = this.openmct.telemetry
-                        .subscribe(this.domainObject, this.updateValues);
-                    this.openmct.telemetry
-                        .request(this.domainObject, {
-                            strategy: 'latest',
-                            size: 1
-                        })
-                        .then(function (values) {
-                            this.updateValues(values[0]);
+                        .subscribe(this.domainObject, function (datum) {
+                            this.updateHistory(datum);
+                            this.updateValues(datum);
                         }.bind(this));
+                    this.requestLad(false);
+                    this.requestHistory(this.openmct.time.bounds());
+                }.bind(this));
+        };
+
+        ImageryController.prototype.requestHistory = function (bounds) {
+            this.requestCount++;
+            this.$scope.imageHistory = [];
+            var requestId = this.requestCount;
+            this.openmct.telemetry
+                .request(this.domainObject, bounds)
+                    .then(function (values) {
+                        if (this.requestCount > requestId) {
+                            return Promise.reject('Stale request');
+                        }
+                        values.forEach(function (datum) {
+                            this.updateHistory(datum);
+                        }.bind(this));
```

Comment:
Instead of binding, you can pass `this` as a second argument to `forEach`.  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach